metpy/constants.py

```diff
@@ -70,6 +71,7 @@
     earth_gravity = g = units.Quantity(1.0, units.gravity).to('m / s^2')
     # Taken from GEMPAK constants
     Re = earth_avg_radius = 6.3712e6 * units.m
+    s = earth_rot_speed = units.Quantity(460, 'm/ s^2')
```

Comment:
`'m / s^2'` to be consistent with other constants units ;)

Reply:
Actually this should probably be `'m / s'` since those are the units listed in the table, and make more sense for something classified as "speed".